source/documentors/references/quick_reference_rst.rst

```diff
@@ -133,6 +133,38 @@ RST can do a lot of things via `directives`_. Here are some common ones:
       `Link to another thing <https://example.com/other>`_
          A brief description of another thing.
 
+Tables
+******
+
+To add a table in RST like the following example:
+
+=====================  =====  ======
+   Outputs                Inputs
+---------------------  -------------
+  A  or  B               A      B
+=====================  =====  ======
+False                  False  False
+True                   False  True
+True                   True   False
+True                   True   True
+=====================  =====  ======
+
+Use the following code:
+
+.. code-block::
+
+  =====================  =====  ======
```

Comment:
I wonder if this format would be difficult to follow, because you have to be precise with the length of the over/under lines. I like the vertical lines we use in tables in the OEP repo, for example, https://raw.githubusercontent.com/openedx/open-edx-proposals/refs/heads/master/oeps/processes/oep-0001.rst

Reply:
What do you think about add the two options to give more freedom to the writers?

Reply:
I think having both will make the document too long. You could consider adding links to "alternative ways of defining tables"

Reply:
Now is ready :)